src/sc_client/client/_payload_factory.py

```diff
@@ -71,10 +72,27 @@ def solve_adj(obj: ScAddr | str):
 
 
 class CreateElementsBySCsPayloadCreator(BasePayloadCreator):
-    def __call__(self, scs: SCsText, *_):
-        if not isinstance(scs, get_origin(SCsText)) or not all(isinstance(n, get_args(SCsText)) for n in scs):
-            raise exceptions.InvalidTypeError("expected object types: List[str]")
-        return scs
+    def __call__(self, scs_list: SCsText, *_):
+        if not isinstance(scs_list, list) or not all(isinstance(n, (str, SCs)) for n in scs_list):
```

Comment:
```suggestion
    def __call__(self, scs_list: List[SCsText], *_):
        if not isinstance(scs_list, list) or not all(isinstance(n, (str, SCs)) for n in scs_list):
```
Update typing, if there is List now

Reply:
Or simply rename _scs_list_ variable to _scs_text_, changing typing is incorrect suggestion